src/NorthwindWebsite/appsettings.json

```diff
@@ -20,5 +20,9 @@
         }
       }
     ]
+  },
+  "FileUploadOptions": {
+    "ImageFileFormats": "jpg, jpeg, bmp, png",
+    "ImageMaxSize": "2000000"
```

Comment:
For JSON you can easily specify number, it will automatically be converted to string on parsing